ketcher-autotests/tests/Structure-Creating-&-Editing/Text-Tool/text-tools-modification-and-manipulation.spec.ts

```diff
@@ -0,0 +1,287 @@
+import { Page, test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  selectLeftPanelButton,
+  selectTopPanelButton,
+  waitForPageInit,
+  pressButton,
+  LeftPanelButton,
+  TopPanelButton,
+  openFileAndAddToCanvas,
+  clickInTheMiddleOfTheScreen,
+  dragMouseTo,
+  selectRing,
+  RingButton,
+  selectDropdownTool,
+  waitForRender,
+} from '@utils';
+import { addTextBoxToCanvas } from '@utils/selectors/addTextBoxToCanvas';
+
+async function createSomeStructure(page: Page) {
+  const point = { x: 97, y: 79 };
+  const point1 = { x: 943, y: 114 };
+  const point2 = { x: 844, y: 579 };
+  const point3 = { x: 66, y: 611 };
+  await page.mouse.move(point.x, point.y);
+  await page.mouse.down();
+  await page.mouse.move(point1.x, point1.y);
+  await page.mouse.move(point2.x, point2.y);
+  await page.mouse.move(point3.x, point3.y);
+  await page.mouse.up();
+}
+
+async function createSomeMovement(page: Page) {
+  const point = { x: 656, y: 359 };
+  const point1 = { x: 906, y: 245 };
+  await page.mouse.move(point.x, point.y);
+  await page.mouse.down();
+  await dragMouseTo(point1.x, point1.y, page);
+  await page.mouse.up();
+}
+
+async function performUndoRedo(page: Page) {
+  await waitForRender(page, async () => {
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+  });
+}
+
+test.describe('Text tools test cases', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Text tool - Modify the created text object', async ({ page }) => {
```

Comment:
I guess "Text tool" in test name is redundant. Current snapshot name is 'Text-tools test-cases-Text-tool-Modify...'. If you remove "Text tool" from test name, snapshot name will be  'Text-tools test-cases-Modify...'